web/server/codechecker_server/api/report_server.py

```diff
@@ -750,22 +753,47 @@ def getRunData(self, run_filter):
 
             results = []
 
-            for instance, tag, cc_version, report_count in q:
+            for run_id, run_date, run_name, duration, tag, cc_version, \
+                report_count \
+                    in q:
+
                 if report_count is None:
                     report_count = 0
 
-                results.append(RunData(instance.id,
-                                       str(instance.date),
-                                       instance.name,
-                                       instance.duration,
-                                       report_count,
-                                       instance.command,
-                                       status_sum[instance.id],
-                                       tag,
-                                       cc_version,
-                                       analyzer_statistics[instance.id]))
+                analyzer_stats = analyzer_statistics[run_id]
+                results.append(RunData(runId=run_id,
+                                       runDate=str(run_date),
+                                       name=run_name,
+                                       duration=duration,
+                                       resultCount=report_count,
+                                       detectionStatusCount=status_sum[run_id],
+                                       versionTag=tag,
+                                       codeCheckerVersion=cc_version,
+                                       analyzerStatistics=analyzer_stats))
             return results
 
+    @exc_to_thrift_reqfail
+    @timeit
+    def getCheckCommand(self, run_history_id, run_id):
+        self.__require_access()
+
```

Comment:
Is it worth to return here early if the provided run_history or run id does not exists in the database?